### PR TITLE
Ensure ec2 metadata API is unavailable on workers

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

**Comment by meatballhat** [comment]
@igorwwwwwwwwwwwwwwwwwwww r?

File: modules/aws_asg/cloud-init.bash [hunk 1]
```diff
@@ -20,6 +20,8 @@ main() {
 
   service travis-worker stop || true
   service travis-worker start || true
+
+  iptables -A OUTPUT -d '169.254.169.254' -j DROP
```
Comment (Lyoness) [inline modules/aws_asg/cloud-init.bash]: ah. I had to look up difference of why `-drop` and not `-reject` :+1: TIL.

**Comment by henrikhodne** [comment]
I don't think packets from Docker containers go through the `OUTPUT` chain in iptables, so I'm not sure if this will work.

tables_traverse

My understanding is that the packet goes PREROUTING -> FORWARD -> POSTROUTING (which is why our current fix for this is `iptables -t nat -I PREROUTING -p tcp -d 169.254.169.254 --dport 80 -j DNAT --to-destination 192.0.2.1`).

**Comment by henrikhodne** [comment]
Something like `iptables -t nat -I PREROUTING -p tcp -d 169.254.169.254 -j DROP` might also work, but we should test to make sure that's the case.

**Pull request merged**